iterations_runners/iterations_runner.c

```diff
@@ -15,11 +15,14 @@
 #include <dlfcn.h>
 #include <err.h>
 #include <string.h>
+#include <inttypes.h>
 
 #define BENCH_FUNC_NAME "run_iter"
+#define TSR_DEFAULT 0xffffffffffffffff
```

Comment:
in the case of doubles we can express negative numbers. I don't see how 0 is better than 0xff.... They are one apart modulo 2^64.  It's kinda moot, as all of the default values are overwritten at the end of the execution.